src/viking.js

```diff
@@ -1,11 +1,48 @@
 // Soldier
-class Soldier {}
-
+class Soldier {
+  constructor(health, strength){
+    this.health = health,
+    this.strength= strength,
+    this.attack = function(){
+      return this.strength;
+    },
+    this.receiveDamage = function(damage){
+      this.health -= damage
+    }
+  }
+}
 // Viking
-class Viking {}
+class Viking extends Soldier{
+  constructor(name, health, strenght){
+    super(health,strenght); 
+    this.name = name;
+  }
+  receiveDamage = function(damage){
+    this.health -= damage
+    if ( this.health <= 0){
+      return `${this.name} has died in act of combat`
+    }else{
+      return `${this.name} has received ${damage} points of damage`
+    }
+  }
+  battleCry = function(){
+    return "Odin Owns You All!"
+  }
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier{
+  receiveDamage = function(damage){
```

Comment:
Aun que herede de soldier te recomiendo hacer uso del constructor para especificar dichas propiedades 👀 
```suggestion
constructor( health, strenght){
    super(health,strenght); 
  }
  receiveDamage = function(damage){
```